src/main/java/com/CoachPlan/CoachPlanController.java

```diff
@@ -21,7 +21,7 @@
 	//(Service stub will be used for backend database calls, think CRUD (create, read update, delete)). 
 	//Also need to come up with a DTO class. Do we want to use different DTO classes for coach and student as well? 
 	@Autowired
-	private ICoachService coachServiceStub;
+	private ICoachService coachService;

```

Comment:
I agree, coachService is a better name, as that is more flexible: it could be either a stub or an implementation.